### PR TITLE
Add Opera GX, eBay Kleinanzeigen, Apple Music + req apps

### PR DESCRIPTION
**Pull request by ninopiamonte**
Add #27 requested apps excluding EMUI apps. Might work on those later

### CODE REVIEW

**Review by paphonb — APPROVED** [review]

**Comment by paphonb** [comment]
Will need to squash before merging

**Comment by ninopiamonte** [comment]
Add Gallery Go, LineageOS Jelly Browser, Tapatalk Pro, XDA

File: app/src/main/res/drawable/apple_music.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+</vector>
```
Comment (patrykmichalik) [inline app/src/main/res/drawable/apple_music.xml:11]:
```suggestion
<vector xmlns:android="http://schemas.android.com/apk/res/android"
    android:width="24dp"
    android:height="24dp"
    android:viewportWidth="24"
    android:viewportHeight="24">
  <path
      android:fillColor="#FF000000"
      android:pathData="m5.5937,17.562a7.2244,7.2244 0,0 1,1.3573 -0.3797c0.3516,-0.0735 0.7044,-0.1421 1.0535,-0.2168A0.8523,0.8523 0,0 0,8.7395 16.0908v-0.1837q0,-5.439 0.0025,-10.8815A1.949,1.949 0,0 1,8.7934 4.5722,0.8502 0.8502,0 0,1 9.4623,3.9377c0.3124,-0.0808 0.6309,-0.1372 0.9481,-0.2021 0.8979,-0.1837 1.7958,-0.3626 2.695,-0.5439l2.7807,-0.5635c0.8207,-0.1641 1.6415,-0.3307 2.4622,-0.4937a7.4363,7.4363 0,0 1,0.8134 -0.1299,0.6039 0.6039,0 0,1 0.6786,0.5904 2.5176,2.5176 0,0 1,0.0147 0.2732q0.0037,3.5096 0,7.0217l0.0012,0.0037v6.9983a3.3,3.3 0,0 1,-0.2989 1.4773,2.5944 2.5944,0 0,1 -1.7003,1.3965 5.6527,5.6527 0,0 1,-1.3108 0.212,2.2919 2.2919,0 0,1 -1.1086,-4.3585 5.2257,5.2257 0,0 1,1.247 -0.3969c0.463,-0.1004 0.9285,-0.1874 1.3891,-0.294a0.7567,0.7567 0,0 0,0.6247 -0.6321,1.1073 1.1073,0 0,0 0.0245,-0.2364q0,-3.335 -0.0024,-6.6676A0.8878,0.8878 0,0 0,18.6888 7.1656,0.3522 0.3522,0 0,0 18.3165,6.8789 4.4333,4.4333 0,0 0,17.7346 6.9598q-1.3965,0.2756 -2.793,0.5586l-4.5323,0.9162c-0.0196,0.0037 -0.0392,0.0122 -0.0588,0.0159A0.5522,0.5522 0,0 0,9.8727 9.0508c-0.0025,0.0514 0,0.1053 0,0.1592q-0.0037,4.7811 -0.0037,9.5622a3.7557,3.7557 0,0 1,-0.2634 1.5031,2.6085 2.6085,0 0,1 -1.7567,1.5104 5.4421,5.4421 0,0 1,-1.3169 0.2107,2.2541 2.2541,0 0,1 -2.352,-1.8914 2.2861,2.2861 0,0 1,1.4136 -2.5431z"
      android:strokeWidth="0.99981"/>
</vector>
```

File: svgs/apple_music.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+</svg>
```
Comment (patrykmichalik) [inline svgs/apple_music.svg:3]:
```suggestion
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<svg
   id="Layer_1"
   data-name="Layer 1"
   viewBox="0 0 24 24"
   version="1.1"
   sodipodi:docname="apple_music.svg"
   inkscape:version="1.1.1 (3bf5ae0d25, 2021-09-20)"
   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
   xmlns="http://www.w3.org/2000/svg"
   xmlns:svg="http://www.w3.org/2000/svg">
  <defs
     id="defs7" />
  <sodipodi:namedview
     id="namedview5"
     pagecolor="#ffffff"
     bordercolor="#666666"
     borderopacity="1.0"
     inkscape:pageshadow="2"
     inkscape:pageopacity="0.0"
     inkscape:pagecheckerboard="0"
     showgrid="false"
     inkscape:zoom="22.892582"
     inkscape:cx="4.1061336"
     inkscape:cy="13.301252"
     inkscape:window-width="2560"
     inkscape:window-height="1015"
     inkscape:window-x="0"
     inkscape:window-y="32"
     inkscape:window-maximized="1"
     inkscape:current-layer="Layer_1" />
  <path
     d="m 5.5937409,17.561982 a 7.2243547,7.2243547 0 0 1 1.3572816,-0.379738 c 0.3515731,-0.07351 0.7043659,-0.142103 1.0534894,-0.216829 A 0.85233774,0.85233774 0 0 0 8.739502,16.090772 v -0.183745 q 0,-5.438965 0.00245,-10.8815489 A 1.948979,1.948979 0 0 1 8.793402,4.5722243 0.85015816,0.85015816 0 0 1 9.4622746,3.9376851 c 0.3123706,-0.080845 0.6308794,-0.1371939 0.9481394,-0.2021215 0.897919,-0.183745 1.795828,-0.362601 2.694977,-0.5438965 l 2.78071,-0.5634927 c 0.820744,-0.1641487 1.641488,-0.330747 2.462232,-0.493666 a 7.4362844,7.4362844 0 0 1 0.813385,-0.1298553 0.60388504,0.60388504 0 0 1 0.678641,0.5904476 2.5175807,2.5175807 0 0 1 0.01471,0.273168 q 0.0037,3.5095819 0,7.0216633 l 0.0012,0.00368 v 6.998348 a 3.2999518,3.2999518 0 0 1 -0.298944,1.477329 2.5943961,2.5943961 0 0 1 -1.700286,1.396484 5.6527439,5.6527439 0 0 1 -1.31075,0.21196 2.2919337,2.2919337 0 0 1 -1.108619,-4.358501 5.2257352,5.2257352 0 0 1 1.247042,-0.396924 c 0.463042,-0.100441 0.928534,-0.187414 1.389136,-0.293994 a 0.75670595,0.75670595 0 0 0 0.624741,-0.63209 1.1073392,1.1073392 0 0 0 0.0245,-0.236425 q 0,-3.335025 -0.0024,-6.6676008 A 0.88783099,0.88783099 0 0 0 18.688849,7.1655714 0.35222295,0.35222295 0 0 0 18.31645,6.8789259 4.433306,4.433306 0 0 0 17.73456,6.9597809 q -1.396494,0.2756176 -2.792968,0.5585937 l -4.532347,0.9162356 c -0.01961,0.00368 -0.0392,0.012248 -0.05881,0.015927 A 0.55222488,0.55222488 0 0 0 9.8726563,9.0507818 c -0.00245,0.05145 0,0.1053499 0,0.1592497 q -0.00367,4.7810895 -0.00368,9.5622395 a 3.755685,3.755685 0 0 1 -0.2633699,1.503054 2.6085034,2.6085034 0 0 1 -1.7566655,1.510412 5.442104,5.442104 0 0 1 -1.3168694,0.2107 2.2540809,2.2540809 0 0 1 -2.3519722,-1.89138 2.2861448,2.2861448 0 0 1 1.4136416,-2.543075 z"
     id="path2"
     style="stroke-width:0.99981" />
</svg>
```

File: svgs/tapatalk_pro.svg [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+    <path d="M20.3171,3.73445a.44747.44747,0,0,0-.39084.19722,1.00955,1.00955,0,0,0-.13927.57775v.119a1.04029,1.04029,0,0,0,.1373.58054.48186.48186,0,0,0,.78528.00754,1.03592,1.03592,0,0,0,.13768-.57935V4.51776a1.028,1.028,0,0,0-.13768-.58371A.44861.44861,0,0,0,20.3171,3.73445Z"/>
+    <path d="M15.90621,3.75945h-.46153v.76664h.44959a.44143.44143,0,0,0,.30435-.09365.34133.34133,0,0,0,.10517-.26864.40223.40223,0,0,0-.10595-.29007A.39441.39441,0,0,0,15.90621,3.75945Z"/>
+  </g>
+</svg>
```
Comment (patrykmichalik) [inline svgs/tapatalk_pro.svg:9]:
```suggestion
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<svg
   id="Layer_1"
   data-name="Layer 1"
   viewBox="0 0 24 24"
   version="1.1"
   sodipodi:docname="tapatalk_pro.svg"
   inkscape:version="1.1.1 (3bf5ae0d25, 2021-09-20)"
   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
   xmlns="http://www.w3.org/2000/svg"
   xmlns:svg="http://www.w3.org/2000/svg">
  <defs
     id="defs17" />
  <sodipodi:namedview
     id="namedview15"
     pagecolor="#ffffff"
     bordercolor="#666666"
     borderopacity="1.0"
     inkscape:pageshadow="2"
     inkscape:pageopacity="0.0"
     inkscape:pagecheckerboard="0"
     showgrid="false"
     inkscape:zoom="32.375"
     inkscape:cx="11.984556"
     inkscape:cy="12"
     inkscape:window-width="2560"
     inkscape:window-height="1015"
     inkscape:window-x="0"
     inkscape:window-y="32"
     inkscape:window-maximized="1"
     inkscape:current-layer="Layer_1" />
  <g
     id="g12"
     transform="matrix(1.0003465,0,0,1.0003465,-0.00418656,-0.00527163)">
    <path
       d="m 20.25483,6.78036 h -5.9216 A 0.32447,0.32447 0 0 1 14.00865,6.45577 V 2.71069 a 0.32448,0.32448 0 0 1 0.32458,-0.3246 h 0.28133 A 9.71727,9.71727 0 0 0 4.07494,17.4581 l -1.45471,3.85738 a 0.50414,0.50414 0 0 0 0.75471,0.59521 l 3.128,-2.11975 A 9.71289,9.71289 0 0 0 20.91274,8.122 9.6177,9.6177 0 0 0 20.25483,6.78036 Z M 8.08945,9.94015 11.50914,5.54906 a 0.24252,0.24252 0 0 1 0.4337,0.1492 v 2.67886 h 2.22492 c 0.2861,0 0.2861,2.12849 0,2.12849 H 8.36563 A 0.35009,0.35009 0 0 1 8.08945,9.94015 Z m 6.56243,8.23736 a 6.9373,6.9373 0 0 1 -3.832,-0.1861 2.9569,2.9569 0 0 1 -1.50071,-2.55268 v -3.884 l 2.62328,-0.53848 v 3.6193 c 0,0 -0.04285,1.1079 0.70992,1.42534 a 3.87385,3.87385 0 0 0 1.73166,-0.0496 4.31181,4.31181 0 0 1 0.26785,2.16622 z"
       id="path2" />
    <path
       d="m 18.06521,3.75945 h -0.40433 v 0.73331 h 0.40635 a 0.41638,0.41638 0 0 0 0.29361,-0.09643 0.346,0.346 0 0 0 0.104,-0.26586 0.36855,0.36855 0 0 0 -0.09839,-0.27181 0.40643,0.40643 0 0 0 -0.30124,-0.09921 z"
       id="path4" />
    <path
       d="m 14.40546,2.7829 v 3.60065 h 6.076 l 0.11505,0.19484 q 0.22977,0.38927 0.42541,0.80076 L 21.78019,6.48394 A 0.07708,0.07708 0 0 0 21.79887,6.43315 V 2.7829 Z m 2.16063,1.944 A 1.02344,1.02344 0 0 1 15.88594,4.93404 H 15.44468 V 5.79472 H 14.94115 V 3.35193 h 0.95312 a 1.10388,1.10388 0 0 1 0.48412,0.1004 0.74594,0.74594 0 0 1 0.32143,0.28609 0.80152,0.80152 0 0 1 0.1123,0.42221 0.70158,0.70158 0 0 1 -0.24603,0.56624 z m 2.49432,1.06782 H 18.52 L 18.06212,4.90031 H 17.66093 V 5.79472 H 17.15779 V 3.35193 h 0.90747 A 1.02554,1.02554 0 0 1 18.73309,3.54478 0.66523,0.66523 0 0 1 18.968,4.09 0.7549,0.7549 0 0 1 18.86008,4.507 0.74,0.74 0 0 1 18.53194,4.77287 l 0.52852,0.99837 z m 2.29673,-1.16623 a 1.47349,1.47349 0 0 1 -0.12738,0.63252 0.962,0.962 0 0 1 -0.365,0.41982 1.07842,1.07842 0 0 1 -1.08646,0.00159 0.971,0.971 0 0 1 -0.36906,-0.417 1.43332,1.43332 0 0 1 -0.13253,-0.62338 V 4.52133 a 1.46507,1.46507 0 0 1 0.13016,-0.6353 0.969,0.969 0 0 1 0.36744,-0.421 1.07694,1.07694 0 0 1 1.08568,0 0.97437,0.97437 0 0 1 0.36744,0.421 1.46635,1.46635 0 0 1 0.12974,0.63371 z"
       id="path6" />
    <path
       d="m 20.3171,3.73445 a 0.44747,0.44747 0 0 0 -0.39084,0.19722 1.00955,1.00955 0 0 0 -0.13927,0.57775 v 0.119 a 1.04029,1.04029 0 0 0 0.1373,0.58054 0.48186,0.48186 0 0 0 0.78528,0.00754 1.03592,1.03592 0 0 0 0.13768,-0.57935 V 4.51776 A 1.028,1.028 0 0 0 20.70957,3.93405 0.44861,0.44861 0 0 0 20.3171,3.73445 Z"
       id="path8" />
    <path
       d="m 15.90621,3.75945 h -0.46153 v 0.76664 h 0.44959 A 0.44143,0.44143 0 0 0 16.19862,4.43244 0.34133,0.34133 0 0 0 16.30379,4.1638 0.40223,0.40223 0 0 0 16.19784,3.87373 0.39441,0.39441 0 0 0 15.90621,3.75945 Z"
       id="path10" />
  </g>
</svg>
```

File: app/src/main/res/drawable/tapatalk_pro.xml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    <path
+        android:fillColor="#000000"
+        android:pathData="M20.3171,3.73445 A0.44747,0.44747,0,0,0,19.9263,3.93167 A1.00955,1.00955,0,0,0,19.787,4.50942 L19.787,4.62842 A1.04029,1.04029,0,0,0,19.9243,5.20896 A0.48186,0.48186,0,0,0,20.7096,5.2165 A1.03592,1.03592,0,0,0,20.8473,4.63715 L20.8473,4.51776 A1.028,1.028,0,0,0,20.7096,3.93405 A0.44861,0.44861,0,0,0,20.3171,3.73445 Z" />
+    <path
+        android:fillColor="#000000"
+        android:pathData="M15.9062,3.75945 L15.4447,3.75945 L15.4447,4.52609 L15.8943,4.52609 A0.44143,0.44143,0,0,0,16.1987,4.43244 A0.34133,0.34133,0,0,0,16.3039,4.1638 A0.40223,0.40223,0,0,0,16.1979,3.87373 A0.39441,0.39441,0,0,0,15.9062,3.75945 Z" />
+</vector>
```
Comment (patrykmichalik) [inline app/src/main/res/drawable/tapatalk_pro.xml:23]:
```suggestion
<vector xmlns:android="http://schemas.android.com/apk/res/android"
    android:width="24dp"
    android:height="24dp"
    android:viewportWidth="24"
    android:viewportHeight="24">
  <path
      android:fillColor="#FF000000"
      android:pathData="m20.2577,6.7774l-5.9237,0A0.3246,0.3246 0,0 1,14.0093 6.4527L14.0093,2.7064a0.3246,0.3246 0,0 1,0.3247 -0.3247l0.2814,0A9.7206,9.7206 0,0 0,4.0722 17.4589l-1.4552,3.8587a0.5043,0.5043 0,0 0,0.755 0.5954l3.1291,-2.1205A9.7163,9.7163 0,0 0,20.9158 8.1195,9.621 9.621,135 0,0 20.2577,6.7774ZM8.0881,9.9383 L11.5089,5.5457a0.2426,0.2426 0,0 1,0.4339 0.1493l0,2.6798l2.2257,0c0.2862,0 0.2862,2.1292 0,2.1292L8.3643,10.504A0.3502,0.3502 0,0 1,8.0881 9.9383ZM14.6528,18.1785a6.9397,6.9397 45,0 1,-3.8333 -0.1862,2.9579 2.9579,0 0,1 -1.5012,-2.5536l0,-3.8853l2.6242,-0.5387l0,3.6206c0,0 -0.0429,1.1083 0.7102,1.4258a3.8752,3.8752 135,0 0,1.7323 -0.0496,4.3133 4.3133,0 0,1 0.2679,2.167z"/>
  <path
      android:fillColor="#FF000000"
      android:pathData="m18.0673,3.7555l-0.4045,0l0,0.7336l0.4065,0a0.4165,0.4165 0,0 0,0.2937 -0.0965,0.3461 0.3461,0 0,0 0.104,-0.266 0.3687,0.3687 0,0 0,-0.0984 -0.2719,0.4066 0.4066,0 0,0 -0.3013,-0.0992z"/>
  <path
      android:fillColor="#FF000000"
      android:pathData="m14.4063,2.7786l0,3.6019l6.0781,0l0.1151,0.1949q0.2298,0.3894 0.4256,0.801L21.7836,6.4809A0.0771,0.0771 45,0 0,21.8022 6.4301L21.8022,2.7786ZM16.5676,4.7233A1.0238,1.0238 45,0 1,15.8873 4.9305L15.4458,4.9305L15.4458,5.7915L14.9421,5.7915L14.9421,3.3478l0.9535,0a1.1043,1.1043 45,0 1,0.4843 0.1004,0.7462 0.7462,0 0,1 0.3215,0.2862 0.8018,0.8018 0,0 1,0.1123 0.4224,0.7018 0.7018,0 0,1 -0.2461,0.5664zM19.0628,5.7915L18.5222,5.7915L18.0642,4.8967L17.6629,4.8967L17.6629,5.7915L17.1595,5.7915L17.1595,3.3478l0.9078,0A1.0259,1.0259 0,0 1,18.7354 3.5407,0.6655 0.6655,0 0,1 18.9704,4.0861 0.7552,0.7552 0,0 1,18.8624 4.5033,0.7403 0.7403,0 0,1 18.5342,4.7693l0.5287,0.9987zM21.3604,4.6248a1.474,1.474 0,0 1,-0.1274 0.6327,0.9623 0.9623,0 0,1 -0.3651,0.42 1.0788,1.0788 0,0 1,-1.0868 0.0016,0.9713 0.9713,0 0,1 -0.3692,-0.4171 1.4338,1.4338 0,0 1,-0.1326 -0.6236L19.2792,4.5176a1.4656,1.4656 0,0 1,0.1302 -0.6355,0.9693 0.9693,0 0,1 0.3676,-0.4211 1.0773,1.0773 0,0 1,1.0861 0,0.9747 0.9747,0 0,1 0.3676,0.4211 1.4669,1.4669 0,0 1,0.1298 0.6339z"/>
  <path
      android:fillColor="#FF000000"
      android:pathData="m20.32,3.7305a0.4476,0.4476 0,0 0,-0.391 0.1973,1.0099 1.0099,0 0,0 -0.1393,0.578l0,0.119a1.0407,1.0407 0,0 0,0.1373 0.5807,0.482 0.482,0 0,0 0.7856,0.0075 1.0363,1.0363 0,0 0,0.1377 -0.5796L20.8503,4.5141A1.0284,1.0284 0,0 0,20.7126 3.9301,0.4488 0.4488,0 0,0 20.32,3.7305Z"/>
  <path
      android:fillColor="#FF000000"
      android:pathData="m15.9075,3.7555l-0.4617,0l0,0.7669l0.4497,0A0.4416,0.4416 0,0 0,16.2 4.4287,0.3414 0.3414,0 0,0 16.3053,4.16 0.4024,0.4024 0,0 0,16.1993 3.8698,0.3945 0.3945,0 0,0 15.9075,3.7555Z"/>
</vector>
```

File: svgs/xda.svg [hunk 5]
```diff
@@ -0,0 +1,9 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+  <g>
+    <polygon points="4.841 9.825 5.947 9.825 5.947 12.604 4.841 12.604 4.841 13.388 6.816 13.388 6.816 9.04 4.841 9.04 4.841 9.825"/>
+    <polygon points="2 13.388 3.195 13.388 3.975 14.96 3.975 12.604 2.869 12.604 2.869 9.825 3.975 9.825 3.975 9.04 2 9.04 2 13.388"/>
+    <path d="M16.36679,9.04042H12.75938v4.3479h3.60741a.79537.79537,0,0,0,.79537-.79537V9.83579A.79537.79537,0,0,0,16.36679,9.04042Zm.04215,3.59468H13.5126V9.79364h2.89634Z"/>
+    <polygon points="10.567 11.165 11.725 9.794 12.362 9.04 11.376 9.04 10.739 9.794 10.073 10.581 9.407 9.794 8.771 9.04 7.785 9.04 8.421 9.794 9.58 11.165 8.338 12.635 7.701 13.388 8.687 13.388 9.324 12.635 10.073 11.749 10.822 12.635 11.459 13.388 12.445 13.388 11.809 12.635 10.567 11.165"/>
+    <path d="M21.20463,9.04042h-2.812a.79538.79538,0,0,0-.79538.79537v3.55253h.75322V11.93041h2.896v1.45791H22V9.83579A.79537.79537,0,0,0,21.20463,9.04042Zm.04176,2.13677h-2.896V9.79364h2.896Z"/>
+  </g>
+</svg>
```
Comment (patrykmichalik) [inline svgs/xda.svg:9]:
```suggestion
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<svg
   id="Layer_1"
   data-name="Layer 1"
   viewBox="0 0 24 24"
   version="1.1"
   sodipodi:docname="xda.svg"
   inkscape:version="1.1.1 (3bf5ae0d25, 2021-09-20)"
   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
   xmlns="http://www.w3.org/2000/svg"
   xmlns:svg="http://www.w3.org/2000/svg">
  <defs
     id="defs17" />
  <sodipodi:namedview
     id="namedview15"
     pagecolor="#ffffff"
     bordercolor="#666666"
     borderopacity="1.0"
     inkscape:pageshadow="2"
     inkscape:pageopacity="0.0"
     inkscape:pagecheckerboard="0"
     showgrid="false"
     inkscape:zoom="32.375"
     inkscape:cx="6.2393822"
     inkscape:cy="14.965251"
     inkscape:window-width="2560"
     inkscape:window-height="1015"
     inkscape:window-x="0"
     inkscape:window-y="32"
     inkscape:window-maximized="1"
     inkscape:current-layer="Layer_1" />
  <g
     id="g12"
     transform="translate(0,0.789)">
    <polygon
       points="4.841,13.388 6.816,13.388 6.816,9.04 4.841,9.04 4.841,9.825 5.947,9.825 5.947,12.604 4.841,12.604 "
       id="polygon2" />
    <polygon
       points="3.975,12.604 2.869,12.604 2.869,9.825 3.975,9.825 3.975,9.04 2,9.04 2,13.388 3.195,13.388 3.975,14.96 "
       id="polygon4" />
    <path
       d="m 16.36679,9.04042 h -3.60741 v 4.3479 h 3.60741 a 0.79537,0.79537 0 0 0 0.79537,-0.79537 V 9.83579 A 0.79537,0.79537 0 0 0 16.36679,9.04042 Z m 0.04215,3.59468 H 13.5126 V 9.79364 h 2.89634 z"
       id="path6" />
    <polygon
       points="7.701,13.388 8.687,13.388 9.324,12.635 10.073,11.749 10.822,12.635 11.459,13.388 12.445,13.388 11.809,12.635 10.567,11.165 11.725,9.794 12.362,9.04 11.376,9.04 10.739,9.794 10.073,10.581 9.407,9.794 8.771,9.04 7.785,9.04 8.421,9.794 9.58,11.165 8.338,12.635 "
       id="polygon8" />
    <path
       d="m 21.20463,9.04042 h -2.812 a 0.79538,0.79538 0 0 0 -0.79538,0.79537 v 3.55253 h 0.75322 v -1.45791 h 2.896 v 1.45791 H 22 V 9.83579 A 0.79537,0.79537 0 0 0 21.20463,9.04042 Z m 0.04176,2.13677 h -2.896 V 9.79364 h 2.896 z"
       id="path10" />
  </g>
</svg>
```

File: app/src/main/res/drawable/xda.xml [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+    <path
+        android:fillColor="#000000"
+        android:pathData="M16.3668,9.04042 L12.7594,9.04042 L12.7594,13.3883 L16.3668,13.3883 A0.79537,0.79537,0,0,0,17.1622,12.5929 L17.1622,9.83579 A0.79537,0.79537,0,0,0,16.3668,9.04042 Z M16.4089,12.6351 L13.5126,12.6351 L13.5126,9.79364 L16.4089,9.79364 Z" />
+    <path
+        android:fillColor="#000000"
+        android:pathData="M10.567,11.165 L11.725,9.794 L12.362,9.04 L11.376,9.04 L10.739,9.794 L10.073,10.581 L9.407,9.794 L8.771,9.04 L7.785,9.04 L8.421,9.794 L9.58,11.165 L8.338,12.635 L7.701,13.388 L8.687,13.388 L9.324,12.635 L10.073,11.749 L10.822,12.635 L11.459,13.388 L12.445,13.388 L11.809,12.635 L10.567,11.165 Z" />
+    <path
+        android:fillColor="#000000"
+        android:pathData="M21.2046,9.04042 L18.3926,9.04042 A0.79538,0.79538,0,0,0,17.5972,9.83579 L17.5972,13.3883 L18.3504,13.3883 L18.3504,11.9304 L21.2464,11.9304 L21.2464,13.3883 L22,13.3883 L22,9.83579 A0.79537,0.79537,0,0,0,21.2046,9.04042 Z M21.2464,11.1772 L18.3504,11.1772 L18.3504,9.79364 L21.2464,9.79364 Z" />
+</vector>
```
Comment (patrykmichalik) [inline app/src/main/res/drawable/xda.xml:23]:
```suggestion
<vector xmlns:android="http://schemas.android.com/apk/res/android"
    android:width="24dp"
    android:height="24dp"
    android:viewportWidth="24"
    android:viewportHeight="24">
  <path
      android:fillColor="#FF000000"
      android:pathData="M4.841,14.177l1.975,0l0,-4.348l-1.975,0l0,0.785l1.106,0l0,2.779l-1.106,0z"/>
  <path
      android:fillColor="#FF000000"
      android:pathData="M3.975,13.393l-1.106,0l0,-2.779l1.106,0l0,-0.785l-1.975,0l0,4.348l1.195,0l0.78,1.572z"/>
  <path
      android:fillColor="#FF000000"
      android:pathData="m16.3668,9.8294h-3.6074v4.3479h3.6074a0.7954,0.7954 0,0 0,0.7954 -0.7954L17.1622,10.6248A0.7954,0.7954 0,0 0,16.3668 9.8294ZM16.4089,13.4241L13.5126,13.4241L13.5126,10.5826h2.8963z"/>
  <path
      android:fillColor="#FF000000"
      android:pathData="M7.701,14.177l0.986,0l0.637,-0.753l0.749,-0.886l0.749,0.886l0.637,0.753l0.986,0l-0.636,-0.753l-1.242,-1.47l1.158,-1.371l0.637,-0.754l-0.986,0l-0.637,0.754l-0.666,0.787l-0.666,-0.787l-0.636,-0.754l-0.986,0l0.636,0.754l1.159,1.371l-1.242,1.47z"/>
  <path
      android:fillColor="#FF000000"
      android:pathData="m21.2046,9.8294h-2.812a0.7954,0.7954 0,0 0,-0.7954 0.7954v3.5525h0.7532v-1.4579h2.896v1.4579L22,14.1773L22,10.6248A0.7954,0.7954 0,0 0,21.2046 9.8294ZM21.2464,11.9662h-2.896L18.3504,10.5826h2.896z"/>
</vector>
```

**Review by patrykmichalik — CHANGES_REQUESTED** [review]
Here are some minor tweaks related to sizing and alignment that I would suggest making.

**Comment by patrykmichalik** [comment]
As per [this comment](https://github.com/LawnchairLauncher/lawnicons/pull/53#issuecomment-967007882), I would suggest linking Jelly to our generic icon for system browsers (`@drawable/browser`). We already use shared icons for dialers, contacts apps, and so on. Sorry for the confusion here.

**Review by patrykmichalik — APPROVED** [review]

**Comment by patrykmichalik** [comment]
Thanks for contributing, @ninopiamonte!

**Pull request merged**